src/constants/index.js

```diff
@@ -1,28 +1,3 @@
+export * from './misc';
 export { default as INDICATORS } from './indicators';
 export { default as CATEGORIES } from './categories';
-
-export const DATA_URL = '/data/smartCitiesData.json';
-
-export const NO_CATEGORY = 'overview';
-export const NO_CITY = 'all-cities';
-
-export const CARD_SIZES = {
-  SMALL: 'small',
-  LARGE: 'large',
-};
-
-export const INDICATOR_CARD_TYPES = {
-  CATEGORY: 'CATEGORY',
-  INDICATOR: 'INDICATOR',
-};
-
-export const COLOR_NAMES = {
-  PRIMARY: 'primary',
-  JOBS: 'jobs',
-  HOUSING: 'housing',
-  INFRASTRUCTURE: 'infrastructure',
-  LIVEABILITY: 'liveability',
-  INNOVATION: 'innovation',
-  PLANNING: 'planning',
-  EXTRA: 'extra',
-};
```

Comment:
I had to move these out to `misc.js`. Otherwise, it wasn't possible to import, say, `COLOR_NAMES` into, say, `INDICATORS`, which was imported through `index.js`.